framework/components/AStepper/AStepper.scss

```diff
@@ -1,5 +1,51 @@
-@import "../../styles";
+@import "../../styles/index.scss";
 
+@include theme(a-steps) using ($theme) {
+  .a-step {
+    &__icon {
+      border: $border-width solid
+        map-deep-get($theme, "steps", "step-icon-border-color");
+    }
+    &__active {
+      .a-step__icon {
+        background: map-get($theme, "base", "color--selected");
+        color: map-get($theme, "base", "inverse-color");
+        border: $border-width solid map-get($theme, "base", "color--selected");
+      }
+    }
+    &__divider {
+      border: 0px;
+      height: 1px;
+      background-color: map-deep-get($theme, "steps", "step-line-color");
+    }
+    &__visited {
+      .a-step__divider {
+        background-color: $color-good;
+        border: 0px;
+        height: 1px;
+      }
+    }
+    &:before {
```

Comment:
Now that we have an element for the line, I don't believe the `:before` line styles need to exist